docs/sources/panels/transformations/types-options.md

```diff
@@ -0,0 +1,285 @@
++++
+title = "Transformation Types and Options"
+type = "docs"
+[menu.docs]
+identifier = "transformation_types_and_options"
+parent = "transformations"
+weight = 300
++++
+
+# Transformation types and options
+
+Grafana comes with the following transformations:
+  
+  - [Reduce](#reduce)
+  - [Merge](#merge)
+  - [Filter by name](#filter-by-name)
+  - [Filter data by query](#filter-data-by-query)
+  - [Organize fields](#organize-fields)
+  - [Join by field (outer join)](#join-by-field-outer-join)
+  - [Add field from calculation](#add-field-from-calculation)
+  - [Labels to fields](#labels-to-fields)
+  - [Group By](#group-by)
+  - [Series to rows](#series-to-rows)
+
+Keep reading for detailed descriptions of each type of transformation and the options available for each, as well as suggestions on how to use them.
+
+## Reduce
+
+Apply a _Reduce_ transformation when you want to simplify your results down to one value. Reduce basically removes time component. If visualized as a table, it reduces a column down to one row (value).
+
+In the **Calculations** field, enter one or more calculation types. Click to see a list of calculation choices. For information about available calculations, refer to the [Calculation list]({{< relref "../calculations-list.md" >}}).
+
+Once you select at least one calculation, Grafana reduces the results down to one value using the calculation you select. If you select more than one calculation, then more than one value is displayed.
+
+Here's an example of a table with time series data. Before I apply the transformation, you can see all the data organized by time.
+
+{{< docs-imagebox img="/img/docs/transformations/reduce-before-7-0.png" class="docs-image--no-shadow" max-width= "1100px" >}}
+
+After I apply the transformation, there is no time value and each column has been reduced to one row showing the results of the calculations that I chose.
+
+{{< docs-imagebox img="/img/docs/transformations/reduce-after-7-0.png" class="docs-image--no-shadow" max-width= "1100px" >}}
+
+## Merge
+
+> **Note:** This transformation is only available in Grafana 7.1+.
```

Comment:
At some point, let's come up with a rule for how long these version notes stick around. Right now, there isn't one.